apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/Brevoppretter.kt

```diff
@@ -135,19 +135,16 @@ class Brevoppretter(
         personerISak: PersonerISak,
     ): Mottaker =
         with(personerISak) {
-            val mottakerFnr: String? =
-                when (verge) {
-                    is Vergemaal -> verge.foedselsnummer.value
-                    is UkjentVergemaal -> null
+            when (verge) {
+                is Vergemaal -> tomMottaker()
+                is UkjentVergemaal -> tomMottaker()
 
-                    else ->
+                else ->
+                    adresseService.hentMottakerAdresse(
+                        sakType,
                         innsender?.fnr?.value?.takeIf { Folkeregisteridentifikator.isValid(it) }
```

Comment:
Vi bruker altså innsender selv om vi finner ForelderVerge (en type Verge på lik linje med de to andre her). Kunne vi kanskje slutte å bruke innsender nå? Dekker ForelderVerge caset hvor innsender har foreldreansvar men ikke forelder-relasjon?